src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,18 @@
+.footer {
```

Comment:
![Screen Shot 2022-12-21 at 7 48 47 PM](https://user-images.githubusercontent.com/118940236/208977424-8a96b07d-0ecd-4ed0-a774-1ca64c5f332d.png)
The same problem as with the header